src/ert/shared/models/base_run_model.py

```diff
@@ -302,13 +302,14 @@ def run_ensemble_evaluator(
             self.ert().resConfig(),
         ).build()
 
+        ensemble.id_ = str(uuid.uuid1()).split("-", maxsplit=1)[0]
```

Comment:
This looks a bit out of place and referring to our discussion can it be shipped as a parameter in the constructor?

Reply:
hence this comment: https://github.com/equinor/ert/pull/3724#discussion_r947745474 ;)

Reply:
Alright, out of place should be fixed!